.bin/create-wasm-app.js

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env node
+
+const { spawn } = require("child_process");
+
+const clone = spawn("git", ["clone", "https://github.com/rustwasm/create-wasm-app", "."]);
+
+clone.on("close", (code) => {
+  if (code !== 0) {
+    console.error()
```

Comment:
FWIW, this was supposed to contain an error message in rust-webpack-template, but I forgot :-/